tests/native/django/test_commands/test_pushtransifex.py

```diff
@@ -133,12 +133,12 @@ def test_template_parsing(mock_find_files, mock_read, mock_push_strings):
 
         # 1.txt
         SourceString(
-            u'This is a short string', 'c1,c2', _tags='t1,t2', _comment="co1",
-            _charlimit=22,
+            u'\nThis is a short string\n', 'c1,c2', _tags='t1,t2',
+            _comment="co1", _charlimit=22,
         ),
         SourceString(
-            u'This is not a shorter string', 'c1,c2', _tags='t1,t2', _comment="co2",
-            _charlimit=33,
+            u'\nThis is not a shorter string\n', 'c1,c2', _tags='t1,t2',
+            _comment="co2", _charlimit=33,
```

Comment:
I have a similar comment later in the PR. The extra `\n`s that were added signify a change in the behavior, which I believe is wrong.

The template code used here:
```
{% t _context="c1,c2" _tags="t1,t2" _comment="co1" _charlimit=22 %}
This is a short string
{% endt %}
```

should detect `This is a short string` as the source string. I believe somewhere in the `do_t()` code a line that stripped new lines was accidentally(?) removed.

Reply:
I think this is the removed code: https://github.com/transifex/transifex-python/pull/9/files#diff-7eaeb1d918a758db0d5ae9b2147918afL79-L82